src/analyzer.ts

```diff
@@ -507,12 +514,12 @@ export class StreamLoader implements UrlLoader {
       throw new Error('Unable to load ${url}.');
     }
 
-    const urlPath = urlObject.pathname;
+    const urlPath = urlObject.pathname || '/';
     const filePath = pathFromUrl(this.config.root, urlPath);
     const file = this._buildAnalyzer.getFile(filePath);
 
     if (file) {
-      return file.contents.toString();
+      return file.contents!.toString();
```

Comment:
My general strategy was to assume that `file.contents` is always defined, because that's the only kind of Vinyl files we support in build.